restonomer-core/src/it/resources/restonomer_context/checkpoints/transformation/checkpoint_add_suffix_to_column_names_transformation.conf

```diff
@@ -0,0 +1,25 @@
+name = "checkpoint_add_suffix_to_column_names_transformation"
+
+data = {
+  data-request = {
+    url = "http://localhost:8080/add-suffix-to-column-names-transformation"
+  }
+
+  data-response = {
+    body-format = "JSON"
+
+    transformations = [
+      {
+        type = "add-suffix-to-column-names"
+        suffix = "_old"
```

Comment:
Here, the undersore should be added by the application itself and not the user.

Reply:
Done! Underscore will be added by the application itself now.